motionplan/motionPlanner.go

```diff
@@ -99,6 +100,46 @@ func (req *PlanRequest) validatePlanRequest() error {
 
 // PlanMotion plans a motion from a provided plan request.
 func PlanMotion(ctx context.Context, request *PlanRequest) (Plan, error) {
+	return planMotionInternal(ctx, request, nil, 0)
+}
+
+// PlanFrameMotion plans a motion to destination for a given frame with no frame system. It will create a new FS just for the plan.
+// WorldState is not supported in the absence of a real frame system.
+func PlanFrameMotion(ctx context.Context,
```

Comment:
Note: This is not an addition, but an artifact of git diffing files improperly due to what used to be the contents of `PlanMotion` moving to `planMotionInternal`.